src/testIntegration/resources/application-test.yml

```diff
@@ -9,7 +9,32 @@ pathfinder.api.base.url: http://localhost:8997
 
 app.db.url: jdbc:hsqldb:mem:data-compliance-db;sql.syntax_pgs=true;shutdown=false
 
+
+data.compliance.request:
+  sqs:
+    provider: embedded-localstack
+    queue.name: data_compliance_request_queue
+    queue.url: http://localhost:4576/queue/data_compliance_request_queue
+    dlq.name: data_compliance_request_dead_letter_queue
+    endpoint.url: http://localhost:4576
+
+data.compliance.response:
+  sqs:
+    provider: localstack
```

Comment:
Why is there a mix of providers for request / response here?

Reply:
This is mistake, thanks for spotting it. It should be 'embedded-localstack'